packages/core/src/command/hl7-notification/hl7-notification-webhook-sender-cloud.ts

```diff
@@ -0,0 +1,36 @@
+import { SQSClient } from "../../external/aws/sqs";
+import { Config } from "../../util/config";
+import { log } from "../../util/log";
+import { capture } from "../../util/notifications";
+import { Hl7Notification, Hl7NotificationWebhookSender } from "./hl7-notification-webhook-sender";
+import { createUuidFromText } from "@metriport/shared/common/uuid";
+
+export class Hl7NotificationWebhookSenderCloud implements Hl7NotificationWebhookSender {
+  private readonly hl7NotificationWebhookSenderQueue: string;
+  private readonly sqsClient: SQSClient;
+
+  constructor(hl7NotificationWebhookSenderQueue: string, sqsClient?: SQSClient) {
+    this.hl7NotificationWebhookSenderQueue = hl7NotificationWebhookSenderQueue;
+    this.sqsClient = sqsClient ?? new SQSClient({ region: Config.getAWSRegion() });
+  }
+
+  async execute(params: Hl7Notification): Promise<void> {
+    const { cxId, patientId, messageReceivedTimestamp } = params;
+    capture.setExtra({
+      cxId,
+      patientId,
+      messageReceivedTimestamp,
+      context: "hl7-notification-webhook-sender-cloud.execute",
+    });
+
+    const payload = JSON.stringify(params);
+    log(
+      `Enqueueing message for processing: ${messageReceivedTimestamp} - cx: ${cxId} - pt: ${patientId}`
+    );
```

Comment:
I don't think this is doing what you're expecting it to do, is it?

The usage is:

```ts
const { log } = out("The prefix w/ cxId and ptId if possible");
...
log("the message");
```

Can bypass "out" and go straight to "log", but:
1. Not the usual pattern, so requires people to look into it to understand (extra overhead); and
2. Harder to enable "debug" whereas in the example above we just add it to the decomposed object as another property beside "log".